cmd/update_test.go

```diff
@@ -141,4 +141,46 @@ var _ = Describe("Update", func() {
 			Eventually(session).Should(gexec.Exit(0))
 		})
 	})
+
+	Describe("When Github returns a 403 error", func() {
+		BeforeEach(func() {
+			command = exec.Command(pathCLI,
+				"update", "check",
+				"--github-api", tempSettings.TestServer.URL(),
+			)
+
+			tempSettings.TestServer.Reset()
+			tempSettings.TestServer.AppendHandlers(
+				ghttp.CombineHandlers(
+					ghttp.VerifyRequest("GET", "/repos/CircleCI-Public/circleci-cli/releases"),
+					ghttp.RespondWith(http.StatusForbidden, []byte("Forbidden")),
+				),
+			)
+		})
+
+		It("should print a helpful error message & exit 255", func() {
+			session, err := gexec.Start(command, GinkgoWriter, GinkgoWriter)
+			Expect(err).ShouldNot(HaveOccurred())
+			Eventually(session).Should(gexec.Exit(255))
+
+			// TODO: This should exit with error status 1, since 255 is a
+			// special error status for: "exit status outside of range".
+			stderr := session.Wait().Err.Contents()
+			Expect(string(stderr)).To(ContainSubstring(`Error: Failed to query the GitHub API for updates.
+
+This is most likely due to GitHub rate-limiting on unauthenticated requests.
+
+To have the circleci-cli make authenticated requests please:
+
+  1. Generate a token at https://github.com/settings/tokens
+  2. Set the token by either adding it to your ~/.gitconfig
+     setting the GITHUB_TOKEN environment variable.
```

Comment:
Missing an `or` in `adding it to your ~/.gitconfig [or] setting the GITHUB_TOKEN environment variable`